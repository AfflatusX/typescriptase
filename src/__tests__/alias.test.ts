import { alias1, alias2, alias3 } from "./examples";

test(
  "alias1",
  async (): Promise<void> => {
    expect(alias1.print({
      name: "alias1",
      path: "src/__tests__/alias.test.ts",
    }))
      .toMatchSnapshot();
  },
);

test(
  "alias2",
  async (): Promise<void> => {
    expect(alias2.print({
      name: "alias2",
      path: "src/__tests__/alias.test.ts",
    }))
      .toMatchSnapshot();
  },
);

test(
  "alias3",
  async (): Promise<void> => {
    expect(alias3.print({
      name: "alias3",
      path: "src/__tests__/alias.test.ts",
    }))
      .toMatchSnapshot();
  },
);
