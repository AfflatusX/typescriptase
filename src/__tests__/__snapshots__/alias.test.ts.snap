// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alias1 1`] = `
"type Ty1 = number;
"
`;

exports[`alias2 1`] = `
"export type Ty2 = string | null;
"
`;

exports[`alias3 1`] = `
"export type Ty3 = Ty2 | Ty1;
"
`;
